TUTORIAL.md

```diff
@@ -1,33 +1,39 @@
-This document is intended for new users of the Workflow feature to learn how to write and understand flows.
+This document is intended for new users of the CloudBees Workflow feature to learn how to write and understand flows.
 
-# Getting started
+# Getting Started
 
-You need to be running Jenkins 1.580.1 or later (1.596.1 or later for more recent features).
-If you have not already done so, make sure Workflow is installed: go to the Plugin Manager and install _Workflow: Aggregator_ and restart Jenkins.
-Also make sure the _Git_ and _JUnit_ plugins are installed and up to date.
+Before you begin, ensure you have the following installed or running:
 
-If you are running CloudBees Jenkins Enterprise 14.11 or later, you already have Workflow (plus some extra associated features).
++ You must be running Jenkins 1.580.1 or later (1.596.1 or later for more recent features).
 
-If you want to play with Workflow without installing Jenkins separately (or touching your production system), try running the [Docker demo](demo/README.md).
++ Ensure Workflow is installed: navigate to **Plugin Manager**, install **Workflow: Aggregator** and restart Jenkins.
 
-# Creating a workflow
++  Git and JUnit plugins are installed and up to date.
```

Comment:
This is actually obsolete, I think. `workflow-aggregator` should automatically pull in sufficiently new `git` and `junit` dependencies.